src/Http/Http.Extensions/src/HttpContextJsonExtensions.Request.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Http.Json
+{
+    public static partial class HttpContextJsonExtensions
```

Comment:
I followed https://github.com/dotnet/aspnetcore/issues/17160 closely but I have some feedback on things I don't like.

Should `ReadFromJsonAsync` and `WriteAsJsonAsync` be on `HttpContextJsonExtensions`? These aren't extensions on HttpContext.

IMO:
* `ReadFromJsonAsync(this HttpRequest, ...)` should be on `HttpRequestJsonExtensions`
* `WriteAsJsonAsync(this HttpResponse, ...)` should be on `HttpResponseJsonExtensions`

Reply:
Sure. It's not a name anyone will see regardless.